packages/dose3/dose3.5.0.1/opam

```diff
@@ -26,7 +26,7 @@ remove: [
 ]
 depends: [
   "ocaml"
-  "ocamlgraph" {>= "1.8.6"}
+  "ocamlgraph" {>= "1.8.6" & <= "1.8.8"}
```

Comment:
Wouldn't it be better to allow the possibility of a 1.8.x patch release and make this:
```suggestion
  "ocamlgraph" {>= "1.8.6" & < "2.0.0"}
```
(and others, if you agree)

Reply:
done